docs/UserGuide.md

```diff
@@ -16,62 +16,72 @@ their contacts into 'work' contacts and 'personal' contacts to better manage the
 
 ## Quick start
 
-1. Ensure you have Java `17` or above installed in your Computer.
+**1. Set up Java**
+- Ensure you have Java `17` or above installed in your Computer.
+- Not sure? Open up Command Prompt (for Windows) or Terminal (for Mac) and type `java -version` to check.
+- Download Java [[here](https://www.oracle.com/java/technologies/javase/jdk17-archive-downloads.html)] if needed.
 
-1. Download the latest `.jar` file from [here](https://github.com/se-edu/addressbook-level3/releases).
+**2. Download UniLink**
+- Download the latest `.jar` file from [here](https://github.com/AY2425S1-CS2103T-W12-3/tp/releases).
 
-1. Copy the file to the folder you want to use as the _home folder_ for your AddressBook.
+**3. Choose a Home Folder**
+- Move the `.jar` file to where you want to store your UniLink data. This will be your 'home folder'.
 
-1. Open a command terminal, `cd` into the folder you put the jar file in, and use the `java -jar addressbook.jar` command to run the application.<br>
-   A GUI similar to the below should appear in a few seconds. Note how the app contains some sample data.<br>
-   ![Ui](images/Ui.png)
+**4. Launch the app!**
+- Open up Command Prompt (for Windows) or Terminal (for Mac) and navigate to the home folder with `cd path/to/your/home/folder`
```

Comment:
Maybe the 'cd path/to/your/home/folder' could be changed to something like 'cd path-to-home-folder'? The multiple slashes may be a bit confusing for some people

Reply:
I think the slashes need to be there cause if not some people may not know that u need to put slashes. The example is there too so I think it's clear? modified slightly